packages/hoprd/src/api/v3/paths/node/info.ts

```diff
@@ -7,7 +7,7 @@ import { ChainConfiguration, health_to_string, Hopr } from '@hoprnet/hopr-utils'
  */
 export const getInfo = async (node: Hopr) => {
   try {
-    const scInfo: ChainConfiguration = node.smartContractInfo()
+    const scInfo: ChainConfiguration = await node.smartContractInfo()
```

Comment:
OK, so this is blocking the removal of the string based contract info... please, consider replacing it with a native object that has addresses instead of strings...